Gemfile

```diff
@@ -42,14 +42,14 @@ supports_windows = true
 
 group :development do
   gem 'puppet-lint',                        :require => false
-  gem 'metadata-json-lint',                 :require => false if ! supports_windows || Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.1')
+  gem 'metadata-json-lint',                 :require => false, :platforms => 'ruby'
```

Comment:
@Iristyle I read your notes in the beaker PR - and it looks like we are in agreement. Bundler looks at every gem (even excluded by without) to determine versions of gems and to resolve conflicts and version dependencies. So 👍